test/tst/demo/core_test.cljs

```diff
@@ -4,7 +4,7 @@
 ;   the root of this distribution.  By using this software in any fashion, you are agreeing to be
 ;   bound by the terms of this license.  You must not remove this notice, or any other, from this
 ;   software.
-(ns tst.demo.core
+(ns tst.demo.core-test
```

Comment:
You don't need the `-test` suffix since there namespace starts with `tst.`  Can we undo this?

Reply:
I believe it's necessary for shadow-cljs to find the test (see [here](https://shadow-cljs.github.io/docs/UsersGuide.html#_testing) regarding the namespace regex)